lib/chat_api/conversations.ex

```diff
@@ -183,6 +184,27 @@ defmodule ChatApi.Conversations do
     Repo.update_all(query, set: [archived_at: DateTime.utc_now()])
   end
 
+  @spec find_freetier_conversations_old_by([{:days, number | Decimal.t()}, ...]) :: Ecto.Query.t()
+  def find_freetier_conversations_old_by(days: days) do
```

Comment:
nitpick -- can we rename this to `query_free_tier_conversations_inactive_for(days: days)`? (`old_by` sounds kinda strange, and i think it's better to start with `query_` rather than `find_` so that it's clear that this is just returning a query and not actually hitting the db)